Source/Engine/Visject/VisjectGraph.cpp

```diff
@@ -371,6 +371,19 @@ void VisjectExecutor::ProcessGroupMath(Box* box, Node* node, Value& value)
         if (value.Type.Type == VariantType::Enum)
             value.AsUint64 = value.AsUint64 | (uint64)tryGetValue(node->GetBox(1), Value::Zero);
         break;
+
```

Comment:
Missing `// Remap` comment to mark the node type impl section.

Reply:
Applied